OpenRA.Mods.Common/Util.cs

```diff
@@ -67,11 +67,26 @@ public static int GetTurnDirection(WAngle facing, WAngle desiredFacing)
 		/// </summary>
 		public static int IndexFacing(WAngle facing, int numFrames)
 		{
+			// 1024 here is the max angle, so we divide the max angle by the total number of facings (numFrames)
 			var step = 1024 / numFrames;
 			var a = (facing.Angle + step / 2) & 1023;
 			return a / step;
 		}
 
+		// Returns the remainder angle after rounding to the nearest whole step / facing
```

Comment:
Missing `<summary></summary>`?

Reply:
Fixed I think? Have never used <summary></summary> before so let me know if it is incorrect.